app/src/main/java/com/example/bigowlapp/model/Attendance.java

```diff
@@ -32,20 +32,20 @@ public void setAuthenticated(boolean authenticated) {
         this.authenticated = authenticated;
     }
 
-    public boolean isAuthAttemptedUserMobileNumber() {
-        return authAttemptedUserMobileNumber;
+    public boolean attemptedAuthByUserMobileNumber() {
+        return attemptedAuthByUserMobileNumber;
```

Comment:
Cause of Case 2.

The reason attemptedAuthByUserMobileNumber is never true in the database is because the database can't access this variable due to mapping issues.

The solution for this is to simply use the standard naming convention for getters and setters (i.e. isAttempted or getAttempted). This can easily be done by deleting this method, and generating the getters via IDE/Android Studio.

![unknown](https://user-images.githubusercontent.com/23643192/111890237-f52b1e00-89bd-11eb-8195-efa578771317.png)

Reply:
Ok, that is something I can fix easily